scripts/migrate-yaml.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+# Copyright 2014-2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# (c) 2016, Matthew Thode <matt.thode@rackspace.com>
+
+"""
+Print out values that have changed, by default just additions and changes.
+
+Additions can be taken as is and placed into an overrides file, but should you should verify you still want to keep it.
+
+Removals are not printed by default because not having an override is the same as taking a default.
+
+Changes are printed, showing what you overrode and what is getting overridden.
+
+CAUTION: danger-mode is only if you KNOW what you are overriding doesn't need to change, warrantee is void if used.
+"""
+
+from __future__ import print_function
+import argparse
+from deepdiff import DeepDiff
+import errno
+import yaml
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--overrides', dest='overrides', required=True,
+                        help='The path to the file with overrides.')
+    parser.add_argument('--info', dest='info', type=bool, default=False,
+                        help='Shows what is NOT overridden.')
+    parser.add_argument('--danger-mode', dest='danger', type=bool, default=False,
+                        help='Print output that is directly dumpable into an overrides file.')
+    return parser.parse_args()
+
+
+def root_cleanup(var):
+    """Cleans up what deepdiff returns so we can have some cleaner output.
+
+    :param var: the key you wish to clean up, just a string
+    :return: the cleaned up string
+    """
+    prefix = 'root[\''
+    suffix = '\']'
+    var = var[var.startswith(prefix) and len(prefix):]
+    var = var[:var.endswith(suffix) and -len(suffix)]
+    var = var.replace('\'][\'', '.')
+    return var
+
+
+def get_config(path):
```

Comment:
Which config are we "getting"? Also would there be a better name or can the docstring be improved to explain this? As it is, "get_config" is vague and "Load yaml data if you can" is about as vague.